src/components/MonsterDetails.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
+import MonsterDetailsStat from './MonsterDetailsStats'
```

Comment:
Was it intentional to change it from MonsterDetailsStats (plural) to MonsterDetailsStat (singular)?